cardano-node/src/Cardano/Node/Tracing/Tracers/Diffusion.hs

```diff
@@ -151,146 +136,127 @@ instance (LogFormatting peer, Show peer) =>
 
 
 docMuxLocal :: Documented (WithMuxBearer (NtN.ConnectionId LocalAddress) MuxTrace)
-docMuxLocal = docMux protoPeerLocal
+docMuxLocal = addDocumentedNamespace  [] docMux'
 
 docMuxRemote :: Documented (WithMuxBearer (NtN.ConnectionId NtN.RemoteAddress) MuxTrace)
-docMuxRemote = docMux protoPeerRemote
+docMuxRemote = addDocumentedNamespace  [] docMux'
 
 
-docMux :: peer -> Documented (WithMuxBearer peer MuxTrace)
-docMux peer = Documented [
+docMux' :: Documented (WithMuxBearer peer MuxTrace)
+docMux' = Documented [
       DocMsg
-        (WithMuxBearer peer
-          MuxTraceRecvHeaderStart)
+        ["RecvHeaderStart"]
         []
         "Bearer receive header start."
     , DocMsg
-        (WithMuxBearer peer
-          (MuxTraceRecvHeaderEnd anyProto))
+        ["RecvHeaderEnd"]
         []
         "Bearer receive header end."
     , DocMsg
-        (WithMuxBearer peer
-          (MuxTraceRecvDeltaQObservation anyProto anyProto))
-        []
-        "Bearer DeltaQ observation."
-    , DocMsg
-        (WithMuxBearer peer
-          (MuxTraceRecvDeltaQSample 1.0 1 1 1.0 1.0 1.0 1.0 ""))
-        []
-        "Bearer DeltaQ sample."
-    , DocMsg
-        (WithMuxBearer peer
-          (MuxTraceRecvStart 1))
+        ["RecvStart"]
         []
         "Bearer receive start."
     , DocMsg
-        (WithMuxBearer peer
-          (MuxTraceRecvEnd 1))
+        ["RecvEnd"]
         []
         "Bearer receive end."
     , DocMsg
-        (WithMuxBearer peer
-          (MuxTraceSendStart anyProto))
+        ["SendStart"]
         []
         "Bearer send start."
     , DocMsg
-        (WithMuxBearer peer
-          MuxTraceSendEnd)
+        ["SendEnd"]
         []
         "Bearer send end."
     , DocMsg
-        (WithMuxBearer peer
-          (MuxTraceState anyProto))
+        ["State"]
         []
         "State."
     , DocMsg
-        (WithMuxBearer peer
-          (MuxTraceCleanExit anyProto anyProto))
+        ["CleanExit"]
         []
         "Miniprotocol terminated cleanly."
     , DocMsg
-        (WithMuxBearer peer
-          (MuxTraceExceptionExit
-              anyProto anyProto anyProto))
+        ["ExceptionExit"]
         []
         "Miniprotocol terminated with exception."
     , DocMsg
-        (WithMuxBearer peer
-          (MuxTraceChannelRecvStart anyProto))
+        ["ChannelRecvStart"]
         []
         "Channel receive start."
     , DocMsg
-        (WithMuxBearer peer
-          (MuxTraceChannelRecvEnd anyProto 1))
+        ["ChannelRecvEnd"]
         []
         "Channel receive end."
     , DocMsg
-        (WithMuxBearer peer
-          (MuxTraceChannelSendStart anyProto 1))
+        ["ChannelSendStart"]
         []
         "Channel send start."
     , DocMsg
-        (WithMuxBearer peer
-          (MuxTraceChannelSendEnd anyProto))
+        ["ChannelSendEnd"]
         []
         "Channel send end."
     , DocMsg
-        (WithMuxBearer peer
-          MuxTraceHandshakeStart)
+        ["HandshakeStart "]
```

Comment:
Seems to be an extra space at the end of string?

Reply:
Thanks for spotting this